src/WordPress/ClientActions.php

```diff
@@ -64,4 +81,33 @@ public function returnWordPressDomain()
 
         return $cf_zones_list;
     }
+
+    private function cacheDomainName($response)
```

Comment:
Lets write tests for this method.

- Sets domain if not in cache
- Updates cache if domain has changed.